rust/src/sip/sip.rs

```diff
@@ -298,6 +392,31 @@ pub unsafe extern "C" fn rs_sip_probing_parser_ts(
     return ALPROTO_UNKNOWN;
 }
 
+#[no_mangle]
+pub unsafe extern "C" fn rs_sip_probing_parser_tcp_ts(
+    _flow: *const Flow,
+    _direction: u8,
+    input: *const u8,
+    input_len: u32,
+    _rdir: *mut u8,
+) -> AppProto {
```

Comment:
This isn't our formatting for new rust code.  Would you be willing to `rustfmt` the whole SIP module, commit, then apply your changes letting rustfmt do its thing?  I think the module is small enough to do a one-shot rustfmt on it, then apply your changes.